docs/user/autoyast.md

```diff
@@ -1,67 +1,37 @@
 # AutoYaST support
 
-Agama offers a mechanism to perform [unattended installations](./unattended). However, we
-would like AutoYaST users to be able to use their profiles in Agama. This document describes how
-Agama could support, to some extent, such profiles.
+We know that many AutoYaST users have invested quite some time in writing their profiles to perform
+unattended installations. For that reason, Agama offers a mechanism to reuse such profiles to some extent.
 
-Bear in mind that it is a living document and our plans could change as we progress.
-
-## What to support
-
-We want to point out that Agama and AutoYaST have different features. Agama is focused on the
+However, Agama and AutoYaST have different features. Unlike AutoYaST, Agama is focused on the
 installation and delegates further configuration to other tools. From this point of view, it is
-clear that many of the sections you can find in an AutoYaST profile will not have an Agama
-counterpart.
+clear that many of the sections in an AutoYaST profile will not have an Agama counterpart.
 
-Nevertheless, we want to cover:
+This document describes how AutoYaST supports work, including the limitations you might find.
 
-- Dynamic profiles, including rules/classes, ERB templates, pre-installation scripts and even "ask
-  lists". See [Dynamic profiles](#dynamic-profiles).
-- Compatibility (partial or full) for the following sections: `networking`, `partitioning`,
-  `language`, `timezone`, `keyboard`, `software`, `scripts`, `users`, `iscsi-client`, `proxy` and
-  `suse_register`. See [Supported sections](#supported-sections).
+:::warning
+AutoYaST support is not fully defined yet, which means that we might add support for more sections
+in the future even if we do not plan to do so now. However, we will do our best to keep this
+document up to date.
 
-We still need to decide how to handle other sections like `firewall`, `bootloader`, `report`,
-`general` or even some elements from `security` or `kdump`.
+Please, let us know if you miss support for any section.
+:::
 
-Finally, we plan to "ignore" many other sections (e.g., all \*-server elements) and sysconfig-like
-elements. See [Unsupported sections](#unsupported-sections).
+## Loading an AutoYaST profile
 
-## Dynamic profiles
+The typical way[^agama-profile-import] of starting an unattended installation in Agama is by setting
+that the URL corresponds to an AutoYaST profile, it will start take care of pre-processing and converting the profile to an Agama equivalent.
```

Comment:
I would be probably more details how exactly specify it. I think currently autoyast=* does not work and user needs to use agama.auto and point it to same url as autoyast= option supports, right?

Reply:
It is linked, but I agree it would be a good idea to 1) support the `autoyast=` option and/or 2) write an example here.

Reply:
I have added a note about using `agama.auto` and put a link to the `boot options` section.